package.json

```diff
@@ -17,5 +17,8 @@
   },
   "scripts": {
     "install": "prades install"
+  },
+  "dependencies": {
+    "@sb/prades": "^0.4.0"
```

Comment:
Thanks for your input, I am aware of the concerns you were raising regarding this approach. This changes would not be proposed for StarVR publishing, so no worries.